sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/parquet/ParquetWrite.scala

```diff
@@ -88,6 +89,15 @@ case class ParquetWrite(
     // Sets compression scheme
     conf.set(ParquetOutputFormat.COMPRESSION, parquetOptions.compressionCodecClassName)
 
+    // Sets Zstd level and Zstd workers, only effect when compression codec is `zstd`
```

Comment:
For clarification, we're already able to set this as a Hadoop property and this PR exposes it as an option under Spark namespace? We shouldn't probably overwrites what users specified here if that's the case.

Reply:
I agree with you, if no one else disagrees, I'll remove the overridden code in `ParquetWrite` later.